utils/firebase/init.ts

```diff
@@ -0,0 +1,34 @@
+import { initializeApp } from "firebase/app";
+import { getAnalytics, isSupported } from "firebase/analytics";
+import { getAuth, GoogleAuthProvider } from "firebase/auth";
+import { collection, getFirestore } from "firebase/firestore";
+
+const apiKey = process.env.NEXT_PUBLIC_FIREBASE_API_KEY;
+const authDomain = process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN;
+const projectID = process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID;
+const appID = process.env.NEXT_PUBLIC_FIREBASE_APP_ID;
+const measurementID = process.env.NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID;
+
+
+const firebaseConfig = {
+    apiKey: apiKey,
+    authDomain: authDomain,
+    projectId: projectID,
+    appId: appID,
+    measurementId: measurementID
+};
+
+
+
+export const app = initializeApp(firebaseConfig);
```

Comment:
Nit: just pass these variables directly into `initializeApp` - imo it looks cleaner and we don't reuse any of these constants.